build-system/server/app.js

```diff
@@ -1230,6 +1230,30 @@ app.get(
   }
 );
 
+if (argv.coverage === 'live') {
```

Comment:
Are there any other preexisting / planned alternative values for `argv.coverage`? If not, would it make sense to simplify `gulp serve` usage by requiring just `--coverage` instead of `--coverage=live` to enable this workflow?

Reply:
When running the server for e2e tests with `--coverage` enabled, it just turns on the coverage-collecting middleware that adds a few reporting endpoints. We wouldn't want to enable on-unload reporting in that case. Open to alternatives if you have suggestions though

Reply:
I see. The reason I found this weird is that we're currently overloading the flag to be both a no-value flag (`--coverage`) and a flag with a value (`--coverage=live`), and I can't think of any precedent for this. I don't have a good alternative to suggest (other than perhaps adding a separate flag for the live part, like `--coverage --live`). Will let you make this call. PR LGTM either way.

Reply:
My thoughts on the above are that, in each case, the second flag is independent from the first (despilte frequently going together); you could run `gulp serve --new_server --esm` or `gulp dist --noconfig --esm` without issue. By contrast, on-unload-reporting would only function when `--compiled` is true, as it's a strict subset of that coverage functionality. The alternative I see would be a separate flag `--live-coverage`, and consuming code would have to test `argv.coverage || argv.liveCoverage`, which strikes me as more error prone. Going to keep this as-is for now, but can always adjust if this becomes unclear/an issue.